pkg/dmidecode

```diff
@@ -7,15 +7,6 @@ sha512=fa31190e43ae7a0b61f6313b45f5eba3ab66390accd587129419e48fca931b631cbb331d6
 pkgver=1
 
 [build]
-cp -f "$K"/config.sub .
-[ -n "$CROSS_COMPILE" ] && \
-  xconfflags="--host=$($CC -dumpmachine) \
-  --with-sysroot=$butch_root_dir"
-
-CPPFLAGS="-D_GNU_SOURCE" CFLAGS="$optcflags" CXXFLAGS="$optcflags" \
-LDFLAGS="$optldflags -Wl" \
-
 sed -i 's,/sbin,/bin,' Makefile
-
```

Comment:
could you look into getting our CFLAGS and LDFLAGS into the build? it would also be great if you could add a desc= tag to vars to describe what the package is about.
thanks!